packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -137,14 +138,13 @@ fragment DebugPage on CloudRun {
 
 const props = defineProps<{
   gql: DebugPageFragment
-  commitsAhead: string
-  commitHash: string
-  runNumber: number
+  commitsAhead: number
 }>()
 
 const debug = computed(() => props.gql)
 
 const relativeCreatedAt = computed(() => dayjs(new Date(debug.value.createdAt!)).fromNow())
+const totalDuration = computed(() => dayjs.duration(debug.value.totalDuration!).format('HH:mm:ss').replace(/^0+:/, ''))
```

Comment:
A comment would be helpful here to describe the regex behavior. Also, may just be better to use a `?? 0` rather than the non-null assertion

Reply:
Also, we're confident cloud will never report back a duration >= 23:59:59? Obviously would be a hung test or something else broken, but this format would roll over if it happened to

Reply:
Comment added.